client/main-layout.jsx

```diff
@@ -82,6 +85,11 @@ const StyledMapsIcon = styled(MapsIcon)`
   height: 36px;
 `
 
+const VersionText = styled(Caption)`
+  margin: 8px 0px 0px 0px;
+  opacity: 0.7;
```

Comment:
Instead of using opacity directly, change the color. We have an exported variable called `colorTextSecondary` in `styles/colors.js` for this particular color